### ISSUE TITLE
Hasiru Dala-Samanvay

### ISSUE THREAD

**Issue by petmongrels** [tech4dev-india/Project-Tracking#11]
**NGO Name**
Hasiru Dala

**Contact Name**
Pradeep T. R.

**Contact Email**
pradeep@hasirudala.in

**Software Partner Name**
Samanvay Foundation

**Lead Name**
Vivek Singh

**Lead Email**	
viveks@samanvayfoundation.org

**Project Description**
https://chintugudiya.org/hasiru-dala-data-management-system/

**Success Metrics**
- Transition to a digital system from excel
- Effective use in the field by the social workers to track their beneficiaries
- Database of waste pickers and family's crucial information (identity, status of social services availed by them)
- Simpler generation of reports

**Start Date**
Mid-Feb 2019

**End Date**
Nov-2019

**Status**

**Evaluator Name**
Kurund

**Estimated Hours**
1900

**Estimated Cost**
2,200,000

**Actual Hours**

**Actual Cost**

**Github URL**
https://github.com/openchs
(Will add implementation repository, current we have worked most on the product features and requirement analysis)

**Google Drive URL**
https://drive.google.com/drive/u/2/folders/1cUntERnumTpFzwuSm19vI7e_rRRBQkhG

**NGO Rating**

**SP Rating**

**Comment by petmongrels** [comment]
Blogs.
1. https://chintugudiya.org/hasiru-dala-data-management-system/
2. https://chintugudiya.org/hasiru-dala-requirement-analysis-and-openchs-missing-features/

**Comment by kurund** [comment]
May 23, 2019: Hasiludala project check-in
Participants: Pradipta, Kurund
* Requirement analysis is completed
* Technology: OpenCHS and Custom development
* Project is divided into 5 phases
* Initially it was planned for Karnataka however, they are starting a new intervention in Tamil Nadu and want to use this project
* Phase 1 is scheduled to be completed by end of July and is on track

**Comment by nubrapika** [comment]
We were not getting any response from our point of contact so we escalated the issue to the leadership team. The following changes were made as a result
1. We have started work on the library system which was planned in a later release. 
2. On Social Security program - We have been given access to the manager (Archana) of all the social workers in Bangalore. Another member of the team acts as a translator as Archana speaks only in Kannada
3. Weekly meetings with the leadership team so there is no further delay. 

On Samanvay's side, development of registration form in Openchs has started as the requirement on it was clear. 

We had two long meetings last week. One on understanding the requirements of Library and the other with some of the senior members on the social security team to understand the gamut of social securities that they provide and the challenges they face
1. Library system: Meeting was held in Samanvay's office on 19 June 2019 with Lakshmi who is Director of the Children's program at HD of which library is a key component. 
Buguri Children’s program has 3 components - there is overlap between Academic Support and Social Security
    1. Academic Support - Hostels, Scholarships, enrolment of dropouts, Loans
    2. Library - Community library, Mobile Library, Events for children
    3. Health/Social Security - Health Camps, bank loans, Aadhar card
Next steps/key decisions from that meeting
- We decided to target a pilot at the end of July 2019 with a couple of libraries
- We are exploring where an open library system called Koha can be customised instead of building a custom app. Lakshmi asked us to evaluate Koha as this is being used by another NGO running community library in a similar setup. 
- The library system will be independent of the social security system. Children who are part of both programs will be registered twice. 

2. Social Security: Meeting was held in HD's office on 22nd June 2019 with Nalini, head of HD and some key members of the social security team. The objective of the meeting was primarily to understand the various types of social security schemes currently given by HD, a master list of eligibility criteria so as to design the decision support in the app to identify the schemes that a waster worker (and his family members) are eligible for. 
We also covered the visit schedule workflow and discussed detailed next steps for an end of July pilot.

**Comment by kurund** [comment]
I had a quick call with Pradipta on Jul 2 and here is the gist :
* Release 1 is deferred for future and is communicated with the NGO
* Based on the discussion with NGO, Release 3: Library work is started.
* Currently, evaluating and almost finalized on Koha:  [http://www.koha.org/](http://www.koha.org/)  for library management
* 3 components of the project
  * OpenCHS mobile application
  * Custom web application - for specific actions
  * Metabase for reporting

End of July deliverables
* Social workers will be primary users of  OpenCHS mobile app and custom web application.
* Implement registration form using OpenCHS mobile app
* Social worker will be able to register Waste Workers
* Custom web application
  * Social worker can download the registration information and print the same
* Setup Library Management system

Pradipta, please feel add/update if I missed anything.

**Comment by nubrapika** [comment]
A few corrections
1. Release one is taking longer and not deferred as the client point of
contact was not responding to our email/questions.
After escalating the issue, the client has changed the point of contact to
avoid further delays and started Koha in parallel (on our recommendation).
Details are in my update.

2. Components of the project.

   - OpenCHS mobile application
   - Custom web application - for specific actions
   - Metabase for reporting
   - *Koha for library management*

Thanks
Pradipta
